pages/common/groups.md

```diff
@@ -1,16 +1,12 @@
 # groups
 
 > Print group memberships for a user.
-> More information: <https://www.gnu.org/software/coreutils/groups>.
+> More information: <https://manned.org/groups.1>.
```

Comment:
This should stay as the gnu.org link, since gnu.org is the original author of the program in question. Same goes for the other tldr pages.